napi-inl.h

```diff
@@ -1645,6 +1645,21 @@ inline Reference<T>& Reference<T>::operator =(Reference<T>&& other) {
 }
 
 template <typename T>
+inline Reference<T>::Reference(const Reference<T>& other) {
+  _env = other.Env();
+  HandleScope scope(_env);
+
+  napi_value value = other.Value();
+  if (value != nullptr) {
+    napi_status status = napi_create_reference(_env, value, 1, &_ref);
```

Comment:
This is always creating a strong reference, even if `other` was a weak reference. I guess it's OK since this should only ever be called by the `Error` class which is always strong. At least add a comment explaining that is the purpose of this constructor.